sklearn/covariance/shrunk_covariance_.py

```diff
@@ -548,6 +548,23 @@ class OAS(EmpiricalCovariance):
     where mu = trace(cov) / n_features
     and shrinkage is given by the OAS formula (see References)
 
+    Examples
+    --------
+    >>> from sklearn.covariance import OAS
+    >>> import numpy as np
+    >>> np.random.seed(42)
```

Comment:
whenever there's a need to make examples deterministic, we pass the seed to the non-deterministic algorithms, instead of setting the global numpy seed, or create a numpy random number generator with a specific seed, and take random values from it.